### DISCUSSION TITLE
Set the primary user of an intune device as the current user as a policy

### DISCUSSION THREAD

**Discussion by SEI-Mkittok** [KelvinTegelaar/CIPP#1799] [Ideas]
We would like the option to set the primary user for the intune device as the last logged in user/current user within CIPP, potentially as a policy. 
Here is a single tenant example script using a CSV
https://github.com/ChanderManiPandey2022/Bulk_Change_Intune_primary_User_Using_Powershell_and_.CSV

**Comment by KelvinTegelaar** [comment]
Only sponsors can make feature requests, this also isn't the place for FRs.
